### ISSUE TITLE
To fix footer in projects page 

### ISSUE THREAD

**Issue by puneetkaur2929** [Vishal-raj-1/Awesome-JavaScript-Projects#1852]
Hey @Vishal-raj-1 
I am interested to revamp footer of projects page.
Kindly assign this issue to me under LGM SOC '21

**Comment by Rizwan-S** [comment]
@puneetkaur2929 sure!

**Comment by 7JKaushal** [comment]
Is this issue still on? I would love to contribute :)

**Comment by puneetkaur2929** [comment]
@Rizwan-S Kindly assign this issue to @7JKaushal because I am unable to clone this repo.

**Comment by Rizwan-S** [comment]
Can you let me know what issue are you facing? @puneetkaur2929

**Comment by puneetkaur2929** [comment]
It's taking too long to clone and stops at 13% almost every time.

**Comment by Rizwan-S** [comment]
You can try to shallow clone the repo. That will fetch only the latest version of the repo and bring down the download size. Hope that helps.

**Comment by puneetkaur2929** [comment]
Ok I ll try.

**Comment by ishikapandey** [comment]
@Rizwan-S is this still open?

**Comment by Rizwan-S** [comment]
Yes it is. However, you need to submit the PR by 12 pm for the PR to be considered in LGMSoC.

**Comment by ishikapandey** [comment]
Ok. Will try.
